webapp/components/DataQuery/Visualizer/Table/Row/Column/ColumnHeader.js

```diff
@@ -27,6 +29,7 @@ const ColumnHeader = (props) => {
 
   const i18n = useI18n()
   const lang = useSurveyLang()
+  const getNodeDefDisplayLabel = useSelector(SurveyFormState.getNodeDefDisplayLabel)
```

Comment:
you cannot use SurveyFormState in DataQuery. add this prop to DataQuery state as well